Lesson_4/1_homework.js

```diff
@@ -0,0 +1,20 @@
+function getObject(num) {
+
+    while(num > 999) {
```

Comment:
Зачем здесь while? Цикл здесь не нужен
```suggestion
    if(num > 999) {
```